pkg/fs/open_windows.go

```diff
@@ -0,0 +1,258 @@
+package fs
```

Comment:
We should add the original copyright notice here. Also for future reference, the go version that these funcs taken from. (Optionally, since these appear to come from different files, mention which ones too?)